src/main/java/com/internet/shop/controller/AddUserController.java

```diff
@@ -0,0 +1,49 @@
+package com.internet.shop.controller;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddUserController extends HttpServlet {
```

Comment:
```suggestion
public class RegisterUserController extends HttpServlet {
```
that's optional, but i think this way it would more clear